independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/Separator.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.resteasy.reactive;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.List;
+
+/**
+ * When used on a {@link List} parameter annotated with {@link RestQuery}, RESTEasy Reactive will split the value of the
+ * parameter (using the value of the annotation) and populate the list with those values.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.PARAMETER })
+public @interface Separator {
```

Comment:
Should it be included in the annotation mapping in the guide? IIRC, we have a table with the Classic -> Reactive mapping.

Reply:
Very good point.

Reply:
Done